### PR TITLE
Fixes: #360, #534, #690, #716: Display default image in case of broken image URL response from twitter servers

### LINKED ISSUE #360: Missing profile picture for various accounts in loklak search
Example - http://loklak.net/search?query=from%3Asinghpratyush_

screenshot from 2017-05-15 14-35-57

This is (probably) because of changing profile picture of people while loklak holds the URL for older picture.

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Added required default image(with corresponding resolutions).
- Made Changes to display default image when the image Url response from twitter image servers is broken.

The URLs returned by api.loklak won't work for sometimes, although they are originally extracted from twitter. But the problem is with twitter image servers, which do not allow to access a lot of images and sometimes none. For more information please see https://twittercommunity.com/t/original-profile-image-not-found/67046. Until a reasonable solution is found on twitter server side, a default image should be placed inside of empty broken image containers with required resolution of the container.

This patch adds default image on broken url response in user-info-box in sidebar(To replace Profile banner image, Profile images of twitter accounts i.e. user, following, and followers).

**Screenshots (if appropriate)** 
Will provide once the api.loklak starts again, currently its down.

**Link to live demo: http://pr-742-fossasia-loklaksearch.surge.sh** 
Parent issue #534, #716 
**Closes #360  #690**

### CODE REVIEW

**Comment by simsausaurabh** [comment]
@singhpratyush @hemantjadon @praveenojha33 please review.

**Comment by praveenojha33** [comment]
Unable to open link even in firefox and chrome incognito
image

**Comment by simsausaurabh** [comment]
@praveenojha33 Please try this link: http://loud-relation.surge.sh

**Comment by praveenojha33** [comment]
@simsausaurabh The problem persist with this link as well 
image
I think it is a temporary problem, I should review it after sometime.

**Comment by praveenojha33** [comment]
No results are coming. Is loklak server down?
image

**Comment by simsausaurabh** [comment]
@praveenojha33 Results are coming, please try again!

**Comment by simsausaurabh** [comment]
@praveenojha33 The count of request response is more on hosted website i.e. Loklak.org. The test PR is deployed on surge and its response capacity is less in comparison to loklak.org. That is why whenever you will search from loklak.org, you will get less broken results in comparision to your local deployment. Its a problem with the surge, which is not able to fetch/hold that match response. And the patch correctly displays default image when there is unavailable image url from twitter. This won't be a problem once the patch gets merged, you will see less default images and more twitter images.

**Comment by praveenojha33** [comment]
Hello @simsausaurabh Now I have compared it with a surge link of previous PR and surge link for this PR . I get this result for surge link of PR 730
image
And for this PR:
image

File: src/app/feed/user-info-box/user-info-box.component.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 	<div *ngIf="(!isUserResponseLoading)" class="wrapper">
 		<div class="image-gallery">
 			<div class="profile-banner-image">
-				<img class="banner" src="{{ apiResponseUser.profile_banner_url }}"/>
+				<img class="banner" src="{{ apiResponseUser.profile_banner_url }}" onError="this.src='./assets/images/default600x200.jpg';"/>
 			</div>
 			<div class="profile-image">
-				<img src="{{ apiResponseUser.profile_image_url }}"/>
+				<img src="{{ apiResponseUser.profile_image_url }}" onError="this.src='./assets/images/def65.jpg';"/>
```
Comment (simsausaurabh) [inline src/app/feed/user-info-box/user-info-box.component.html]: @praveenojha33 The default image gets load only when there is any error(like broken URL). I have not replaced the original response url. As mentioned in the PR description, the problem is with twitter image servers, which do not allow to access a lot of images and sometimes none. And it also depends on host response capacity.

**Comment by mariobehling** [comment]
I am not sure, if this is the best approach, cause it does not start to path a way to solve the issue. A better approach would seem to me to implement the solution on the server itself.

**Comment by simsausaurabh** [comment]
I looked into the current available solutions. But still I'll try to work on to resolve this on server side. Thanks @mariobehling :+1:

**Comment by mariobehling** [comment]
Why are you committing this to master? Commit to dev, please.

**Pull request closed without merging**